dbt/models/default/default.vw_pin_status.sql

```diff
@@ -0,0 +1,160 @@
+                    AND (
+                        com.parid IS NOT NULL
+                        OR (
+                            oby.class IS NOT NULL
+                            AND (SUBSTR(oby.class, 1, 1) NOT IN ('1', '2'))
+                            AND oby.class NOT IN ('OA2', 'EX')
+                        )
+                    )
+                    THEN TRUE
+                    -- If a parcel is class 2 in oby and exists in comdat, then
+                    -- it's mixed use.
+                WHEN
+                    (SUBSTR(oby.class, 1, 1) = '2' OR oby.class = 'OA2')
+                    AND com.class IS NOT NULL
+                    THEN TRUE
+                ELSE FALSE
+            END
+        ) AS is_mixed_use
+    FROM {{ source('iasworld', 'pardat') }} AS par
+    LEFT JOIN {{ source('iasworld', 'comdat') }} AS com
+        ON par.parid = com.parid
+        AND par.taxyr = com.taxyr
+        AND com.cur = 'Y'
+        AND com.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'dweldat') }} AS dwel
+        ON par.parid = dwel.parid
+        AND par.taxyr = dwel.taxyr
+        AND dwel.cur = 'Y'
+        AND dwel.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'oby') }} AS oby
+        ON par.parid = oby.parid
+        AND par.taxyr = oby.taxyr
+        AND oby.cur = 'Y'
+        AND oby.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'land') }} AS land
+        ON par.parid = land.parid
+        AND par.taxyr = land.taxyr
+        AND land.cur = 'Y'
+        AND land.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'aprval') }} AS aprval
+        ON par.parid = aprval.parid
+        AND par.taxyr = aprval.taxyr
+        AND aprval.cur = 'Y'
+        AND aprval.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'exadmn') }} AS admn
+        ON par.parid = admn.parid
+        AND par.taxyr = admn.taxyr
+        AND admn.cur = 'Y'
+        AND admn.deactivat IS NULL
+    WHERE par.cur = 'Y'
+        AND par.deactivat IS NULL
+    GROUP BY par.parid, par.taxyr
+),
+
+-- These CTEs make it easier to work with CDUs from oby, comdat, dweldat since
+-- those tables aren't unique by parid and taxyr.
+dwel_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'dweldat'),
+    cdu_column = 'cdu'
+    ) }}),
+
+com_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'comdat'),
+    cdu_column = 'user16'
+    ) }}),
+
+oby_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'oby'),
+    cdu_column = 'user16'
+    ) }})
+
+SELECT
+    pdat.parid AS pin,
+    pdat.class,
+    pdat.taxyr AS year,
+    colo.is_corner_lot,
+    ahsap.is_ahsap,
+    COALESCE(vpe.pin IS NOT NULL, FALSE) AS is_exempt,
+    COALESCE(pin.tax_bill_total = 0, NULL) AS is_zero_bill,
+    vpcc.is_parking_space,
+    vpcc.parking_space_flag_reason,
+    vpcc.is_common_area,
+    SUBSTR(pdat.parid, 11, 1) = '8' AS is_leasehold,
+    ahsap.is_mixed_use,
+    pdat.class = 'RR' AS is_railroad,
```

Comment:
Just for my own sanity, do a quick check and make sure that we don't also need to use the `ROLLTYPE` column or similar to flag railroad PINs. I seem to remember something about that from awhile back.